examples/sec-sentiment-analysis/sec-sentiment-analysis.ipynb

```diff
@@ -0,0 +1,815 @@
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "30b00cdc",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████████████████████████████████| 2499/2499 [01:22<00:00, 30.37it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "annotations = []\n",
+    "for element in tqdm.tqdm(elements):\n",
+    "    inference = sentiment_pipeline(element.text, truncation=True)\n",
+    "    result = [LabelStudioResult(\n",
+    "              type=\"choices\",\n",
+    "              value={\"choices\": [inference[0][\"label\"].title()]},\n",
+    "              from_name=\"sentiment\",\n",
+    "              to_name=\"text\",\n",
+    "    )]\n",
+    "    annotations.append([LabelStudioAnnotation(result=result)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7db2e8ae",
+   "metadata": {},
+   "source": [
+    "The `stage_for_label_studio` function formats the outputs for upload to LabelStudio. After we save the results as a JSON, we can create a new project in LabelStudio and upload the training examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "773e7770",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "label_studio_data = stage_for_label_studio(\n",
+    "    elements=elements,\n",
+    "    annotations=annotations,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "ca752c3f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "6e281376",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"sec-sentiment-analysis.json\", \"w\") as f:\n",
+    "    json.dump(label_studio_data, f, indent=4)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1cdd8ddc",
+   "metadata": {},
+   "source": [
+    "### Train a Sentiment Model <a id=\"train\"></a>\n",
+    "\n",
+    "After labeling the data, we're ready to train the sentiment analysis model using the Huggingface `transformers` library. Check out the [Huggingface documentation](https://huggingface.co/blog/sentiment-analysis-python) for more information on how to train models in `transformers`.\n",
+    "\n",
+    "The first step is to export the labeled data from LabelStudio. When you export the data, select the JSON-Min format. Once that's done, we'll convert the dictionary to a Huggingface `Dataset` object so that it can be used in the model training pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "3909505f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datasets import Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "c30b6171",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"sec-sentiment-analysis-labeled.json\", \"r\") as f:\n",
```

Comment:
I get to this cell and it can't find this file. Is this supposed to have the name `sec-sentiment-analysis.json`, or am I supposed to jump over to Label Studio and do create this file somehow?

Reply:
Actually never mind, I see the instructions now.